models/equflash/calc_cps.py

```
@@ -0,0 +1,74 @@
+    actual_false = df_uniq["e_above_hull_mp2020_corrected_ppd_mp"] > 0
+    TP = model_true & actual_true
+    FN = actual_true & model_false
+    FP = actual_false & model_true
+    TN = actual_false & model_false
+    n_true_pos = TP.sum()
+    n_false_pos = FP.sum()
+    n_false_neg = FN.sum()
+    n_true_neg = TN.sum()
+    n_total_pos = n_true_pos + n_false_neg
+    n_total_neg = n_true_neg + n_false_pos
+    prevalence = n_total_pos / (n_total_pos + n_total_neg)
```

Ruff (0.14.1)

Local variable `prevalence` is assigned to but never used

Remove assignment to unused variable `prevalence`

(F841)